src/lt/macros.clj

```diff
@@ -47,30 +47,30 @@
     [(first body) (rest body)]
     [[] body]))
 
-(defmacro on [name & body]
+(defmacro ^:private on [name & body]
```

Comment:
Ok that makes sense. We probably should try to narrow down the public api quite a lot to make it maintainable (and a lot easier to make changes in the future without to much fear of breaking stuff).